app/views/clears-data.html

```diff
@@ -0,0 +1,34 @@
+
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Clear data? | GOV.UK Prototype Kit
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ govukBackLink({
+          "text": "Back",
+          "href": "javascript: window.history.go(-1)"
+        }) 
+      }}
+
+      <h1 class="govuk-heading-xl">
+        Are you sure?
+      </h1>
+      {{ govukWarningText({
+        text: "This will clear all of the data entered in this session",
+        iconFallbackText: "Warning"
+      }) }}
+
+
+      {{ govukButton({
+        text: "Clear data",
+        href: "/prototype-admin/clear-data"
```

Comment:
Non-blocking:

I'd expect this kind of interaction to be a `<form>` that POSTs to an endpoint, rather than a link to a new page.